infra/recipes/terraform/azure/modules/mysql/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "azurerm_subnet" "mysql_subnet" {
+  name = var.subnet_name
+  resource_group_name = var.resource_group_name
+  virtual_network_name = var.virtual_network_name
+  address_prefixes = ["10.0.0.0/24"]
```

Comment:
Shall we move the address to variable as this can change?

Reply:
Yes